draft-ietf-tls-tls13.md

```diff
@@ -3325,11 +3325,12 @@ certificate_list
 
 extensions:
 : A set of extension values for the CertificateEntry. The "Extension"
-  format is defined in {{extensions}}. Valid extensions include
-  OCSP Status extension ({{RFC6066}}) and
-  SignedCertificateTimestamps ({{!RFC6962}}).  An extension MUST only be present
-  in a Certificate message if the corresponding
-  ClientHello extension was presented in the initial handshake.
+  format is defined in {{extensions}}. Valid extensions for server certificates
+  include OCSP Status extension ({{RFC6066}}) and
+  SignedCertificateTimestamps ({{!RFC6962}}). Extensions in the Certificate
+  message from the server MUST correspond to one from the ClientHello message,
+  extensions in the Certificate from the client MUST correspond with an
+  extension in the CertificateRequest message from the server.
```

Comment:
That's a comma splice (switch to semicolon before "extensions in the Certificate from the client MUST [...]" to fix).